dune/grid/cpgrid/processEclipseFormat.cpp

```diff
@@ -99,12 +103,52 @@ namespace cpgrid
         std::vector<double> coordData;
         ecl_grid.exportCOORD(coordData);
 
-        std::vector<double> zcornData;
-        ecl_grid.exportZCORN(zcornData);
-
         std::vector<int> actnumData;
         ecl_grid.exportACTNUM(actnumData);
 
+        std::vector<double> zcornData;
+        {
```

Comment:
> It would be good to have this section as a separate function returning (possibly sanitized) `zcornData`,

I don't disagree.  Having to pass **both** the `ecl_grid` and the `actnumData` is a bit annoying though.  Maybe we could add a function
```C++
struct CpGridInput {
    std::vector<int>    actnum;
    std::vector<double> coord;
    std::vector<double> zcorn;
};

CpGridInput getCpGridKeywords(const EclipseGrid& G);
```
to `processEclipseFormat.cpp` but that is essentially just moving the problem around.